server/config.py

```diff
@@ -48,10 +48,11 @@
 CHALLONGE_KEY = "challonge_key"
 CHALLONGE_USER = "challonge_user"
 
-API_CLIENT_ID = os.getenv("API_CLIENT_ID", "6ccaf75b-a1f3-48be-bac3-4e9ffba81eb7")
+API_CLIENT_ID = os.getenv("API_CLIENT_ID", "client_id")
```

Comment:
why?

Reply:
Client_id has changed, so the ID that was there is no longer relevant anyway